src/server/services/wms/qgswmsgetcapabilities.cpp

```diff
@@ -786,10 +786,6 @@ namespace QgsWms
     // Keyword list
     addKeywordListElement( project, doc, layerParentElem );
 
-    // Metadata (empty but needed for OGC tests RECOMMENDATIONS)
```

Comment:
@sbrunner As indicated in the comment, the field `MetadataURL` is necessary to pass OGC tests (and then obtain the OGC certification for WMS 1.3.0, on which we're currently working on).  

These tests are run nightly on the Oslandia gitlab (with master branch) and a HTML report is published then : http://test.qgis.org/ogc_cite.

So it should NOT be removed. However, some reflection could be done to provide a way to configure this tag in some way. What do you think?

Reply:
First I Think that's relay strange that the certification verify that the capabilities is valid with the schema..., After that yes we can provide a way to fill those fields :-)

Reply:
> First I Think that's relay strange that the certification verify that the capabilities is valid with the schema..

Yes maybe, I don't know :). 

FYI, it's the next test which  checks this tag : http://test.qgis.org/ogc_cite/2018_02_23_00_57_wms_1_3_0.html#a17e1fcc-95b3-4c92-962f-229980bc8ed5/d1e17382_1/d1e11300_1

> But in the WMS 1.3.0 Specification. In the Table 7 and in the start of the page 66 (schema) it's marked ad optional
> Then I can't parse it with OWSLib neither in version 1.1.1 neither in version 1.3.0

Is it an issue in OWSLib? Because it should be able to parse the document, even if it's indicated as an optional tag in the WMS 1.3.0 documentation...

Reply:
The test is labeled 'Every named Layer in the capabilities document contains a MetadataURL element' but on this tag we have an empty Title Abstract and Name shouldn't  we also remove them?
---
The real issue it's that the type is required but we don't provide it, ans OWSlib don't like this :-)

Reply:
I just see that we can have a name than we should be able to also fill this tag...

Reply:
OK, I leave you to investigate so :). 

Let me know when you reach a state where OGC tests still pass and GetCapabilities document may be parsed by OWSLib.